README.md

```diff
@@ -36,55 +36,120 @@ Then install the library:
 
 ## Getting started
 
-### Start Server
+### 1. Start server (aka. remote end)
 
-The required server is the `selenium-server-standalone-#.jar` file provided here: http://selenium-release.storage.googleapis.com/index.html
+To control a browser, you need to start a *remote end* (server), which will listen to the commands sent
+from this library and will execute them in the respective browser.
 
-Download and run the server by **replacing #** with the current server version. Keep in mind **you must have Java 8+ installed to run this command**.
+This could be Selenium standalone server, but for local development you can send them directly to so called "browser driver" like Chromedriver or Geckodriver.
 
-    java -jar selenium-server-standalone-#.jar
+#### a) Chromedriver
 
-### Create a Browser Session
+Install latest Chrome and [Chromedriver](https://sites.google.com/a/chromium.org/chromedriver/downloads).
+Make sure to have compatible version of Chromedriver and Chrome!
 
-When creating a browser session, be sure to pass the url of your running server.
+Run `chromedriver` binary, you can pass port argument, so that it listens on port 4444:
 
-```php
-// This would be the url of the host running the server-standalone.jar
-$host = 'http://localhost:4444/wd/hub'; // this is the default url and port where Selenium server starts
+```sh
+chromedriver --port=4444
 ```
 
-##### Launch Chrome
+#### b) Geckodriver
 
-Install latest Chrome and [Chromedriver](https://sites.google.com/a/chromium.org/chromedriver/downloads).
+Install latest Firefox and [Geckodriver](https://github.com/mozilla/geckodriver/releases).
+Make sure to have compatible version of Geckodriver and Firefox!
 
-The `chromedriver` binary must be placed in system `PATH` directory, otherwise you must provide the path when starting Selenium server
-(eg. `java -Dwebdriver.chrome.driver="/path/to/chromedriver" -jar selenium-server-standalone-#.jar`).
+Run `geckodriver` binary (it start to listen on port 4444 by default):
 
-```php
-$driver = RemoteWebDriver::create($host, DesiredCapabilities::chrome());
+```sh
+geckodriver
```

Comment:
You specified the default port for chromedriver can you do it for geckodriver?

Reply:
Geckodriver has 4444 as the default (while chromedriver does not).

Reply:
Okay